### ISSUE TITLE
Webserver configs php-fpm version

### ISSUE THREAD

**Issue by efeefe** [getgrav/grav#1936]
Current [sample webserver config](https://github.com/getgrav/grav/blob/develop/webserver-configs/nginx.conf) for nginx assumes the webserver is using php-fpm5.0. In recent OSs, default version is already 7.0. Perhaps the sample config should have both, commented out, and ask user to choose.

**Comment by rhukster** [comment]
True, it's only an example. If you want to provide an updated one, that would be appreciated.
